src/main/java/net/simpvp/Misc/ServerListMessage.java

```diff
@@ -0,0 +1,55 @@
+package net.simpvp.Misc;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.List;
+import java.util.concurrent.ThreadLocalRandom;
+
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.server.ServerListPingEvent;
+
+import net.md_5.bungee.api.ChatColor;
+
+
+public class ServerListMessage implements Listener {
+	
+	List<?> listOfMOTDS = Misc.instance.getConfig().getList("serverListMessage");
+	String motd = "Welcome to simpvp";
+	static Class<?> eventsClass;
+	static Field field;
+	static Method method;
+	String eventName = null;

```

Comment:
Let's make most of these non-static, and move the initialization from a function (`initEvents`) to a constructor, similar to this class https://github.com/C4K3/Misc/blob/ed3c8d499e0bbb61f6304d4ad7da1332ac7d79a7/src/main/java/net/simpvp/Misc/DamageSource.java#L21-L24

The constructor should also take the plugin as input, and use that to get the config, rather than reading it globally.